addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml

```diff
@@ -4,10 +4,11 @@
           xmlns:ask="http://schemas.android.com/apk/res-auto"
           android:keyWidth="10%p"
           android:keyHeight="@integer/key_normal_height">
+
     <Row>
         <Key android:codes="q" android:popupCharacters="1¹₁" ask:hintLabel="1" android:keyEdgeFlags="left"/>
         <Key android:codes="w" android:popupCharacters="2²₂ŵ" ask:hintLabel="2"/>
-        <Key android:codes="e" android:popupCharacters="3³₃èéêëęē€" ask:hintLabel="3"/>
+        <Key android:codes="e" android:popupCharacters="3€³₃èéêëęē" ask:hintLabel="3"/>
```

Comment:
for consistency, I'd like to keep this character in the end. We have an on-going discussion on re-ordering.

Reply:
Ok. The reason for that was to have a more useful caracter (€) easier to reach. 
Would it work putting it int the forder xml-it? We europeans use € not $ ;)
I can change it back, no prob

Reply:
you can add a `currency_data` file for `it` locale. This is the data we have for Hebrew (`iw` locale): https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/ime/app/src/main/res/values-iw/currency_data.xml

You'll need to create such file under `ime/app/src/main/res/values-it` for Italian locale. Or any other locale you wish.

Reply:
done